content/docs/rfcs/14/README.md

```diff
@@ -38,6 +38,10 @@ The `timestamp` field MAY be filled out to signify the time at which the message
 This field holds the Unix epoch time in nanoseconds. 
 Omitting it means the timestamp is unspecified.
 
+The `ephemeral` field MAY be filled out to signify the transient nature of the message.
+Omitting it means the message is permananent, and must be added to the store.
```

Comment:
I wouldn't say `permanent`. It just indicates that the message COULD be stored by store nodes. If this message SHOULD be stored, this field MUST be set to `false`. (Since `false` is the default for `bool` fields in proto3, omitting it will have the same effect.)

Reply:
Nice catch! Amending!

Reply:
Addressed in [eeed3e2](https://github.com/vacp2p/rfc/pull/532/commits/eeed3e28fef460452ea7a4c58084d6ad7cf3f2e7)